kubernetes/op-scim-deployment.yaml

```diff
@@ -16,9 +16,20 @@ spec:
       - name: op-scim
         image: 1password/scim:v1.6.0
         command: ["/op-scim/op-scim"]
-        args: ["--session=/secret/scimsession", "--letsencrypt-domain={YOUR-DOMAIN-HERE}"]
+        args:
+        - "--session=/secret/scimsession"
+        - "--letsencrypt-domain={YOUR-DOMAIN-HERE}"
+        - "--redis-host=redis
```

Comment:
I assume you are adding these default redis config args as a form of documentation?

Reply:
exactly